src/oopgroop/projects/boggIe/api/WordFileProcessor.java

```diff
@@ -0,0 +1,22 @@
+package oopgroop.projects.boggIe.api;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class WordFileProcessor {
+    public static String[] ConvertFileToWordList(File wordFile) throws FileNotFoundException {
+        Scanner sc = new Scanner(wordFile);
```

Comment:
This won't affect our project at all (probably), but typically you should close `Scanner`s after you're done using them.

Reply:
I don't really know the difference between the "Add single comment" button and "Start a review" button, so I clicked add a single comment.